### ISSUE TITLE
error: portal is not a recognized basename of an API endpoint on the server

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#6093]
### Observed behavior

JS error when loading facility page:

![image](https://user-images.githubusercontent.com/2367265/69152885-3b5f2d00-0aab-11ea-8b8d-d940fa2388e0.png)

### Expected behavior

no errors

### User-facing consequences

unknown

### Errors and logs

```
kolibri/core/assets/src/api-resource.js] portal is not a recognized basename of an API endpoint on the server
Resource @ api-resource.js:599
```

### Steps to reproduce

load 'facility' app

### Context

0.13.0 alpha 3

**Comment by ralphiee22** [comment]
I think this is an error related to running in dev mode, because I have seen this happen on kdp. @rtibbles if you can confirm

**Comment by rtibbles** [comment]
This is due to the not really used and slightly buggy schema inspection mechanisms. I think this is not important, and does not need to be fixed in the scope of 0.13.0.

I have some WIP to remove this completely, which I think we can do as a patch fix.

**Comment by rtibbles** [comment]
The code that makes these warnings has been removed, so this issue is fixed.
